ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -210,3 +206,27 @@ def transform_date_for_solr(date):
         log.error("Exception {} occured on date transformation of {}"
                   .format(e, date))
         return None
+
+
+def get_localized_date(value):
+    """Take an isoformat date and return a localized date, e.g.
+    '24. Juni 2020'.
+    """
+    try:
+        dt = isodate.parse_datetime(value)
+        if isinstance(dt, datetime):
+            return localised_nice_date(dt, show_date=True, with_hours=False)
+    except Exception:
+        return None
+
+
+def get_date_picker_format(value):
+    """Take an isoformat date and return a date in the date-picker format,
+     e.g. '24.06.2020'.
+    """
+    try:
+        dt = isodate.parse_datetime(value)
+        if isinstance(dt, datetime):
+            return isodate.strftime(dt, DATE_PICKER_FORMAT)
+    except Exception:
+        return ""
```

Comment:
in general looks nice to me
I am just asking myself how easy is it to debug in case we got an exception.
Would it make sense to add some statement?

Reply:
Sure, I can add a debug statement.